### ISSUE TITLE
Cover indicators aren't showing properly

### ISSUE THREAD

**Issue by Zamrod** [Aedif/tactical-grid#23]
I've run a bunch of tests using Tactical Grid 1.22.0 and using any combination of Alternate Token Cover, Alternate Token Visibility, Midi, and/or Simbul's Cover Calculator. This is in DnD5e 2.4.1 using Foundry 11.315.

The cover indicator doesn't appear at all unless I select Simbul's Cover Calculator. Then it appears on some tokens but not others and it doesn't appear to be correct as to which ones have cover.

If I set it to use Alternate Token Cover, it gives constant errors in the console of:

```
asurer.js:289 Error: This is not a registered game setting
    at ClientSettings.get (foundry.js:3339:42)
    at Settings.get (ModuleSettingsAbstract.js:56:33)
    at CoverCalculator.calcConfiguration (CoverCalculator.js:112:35)
    at new CoverCalculator (CoverCalculator.js:95:34)
    at computeCoverBonus (utils.js:229:24)
    at DistanceMeasurer.drawLabels (measurer.js:287:21)
    at DistanceMeasurer.showMeasures (measurer.js:34:22)
    at 🎁CONFIG.Canvas.rulerClass.prototype.measure#aedifs-tactical-grid (settings.js:509:30)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:616:16)
    at 🎁CONFIG.Canvas.rulerClass.prototype.measure#0 (libWrapper-wrapper.js:189:20)
    at DragRulerRuler.scheduleMeasurement (foundry_imports.js:153:8)
    at DragRulerRuler.onMouseMove (foundry_imports.js:145:22)
    at Token5e.onEntityLeftDragMove (main.js:159:37)
    at Token5e.onEntityLeftDragMoveSnap (main.js:153:23)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:616:16)
    at 🎁Token.prototype._onDragLeftMove#0 (libWrapper-wrapper.js:189:20)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:507:22)
    at Token5e._onDragLeftMove (Token.js:310:3)
    at 🎁CONFIG.Token.objectClass.prototype._onDragLeftMove#0 (libWrapper-wrapper.js:187:52)
    at MouseInteractionManager.callback (foundry.js:31766:17)
    at #handleDragMove (foundry.js:32113:26)
    at #handleMouseMove (foundry.js:32085:37)
    at Na.notifyListeners (earcut.js:128:75)
    at Na.notifyTarget (earcut.js:128:75)
    at Na.propagate (earcut.js:128:75)
    at Na.dispatchEvent (earcut.js:128:75)
    at Na.mapPointerMove (earcut.js:128:75)
    at Na.mapEvent (earcut.js:128:75)
    at yn.onPointerMove (earcut.js:128:75)
```

**Comment by Aedif** [comment]
Use the latest versions of these modules. I think this is an error that was resolved by Token Cover some weeks ago.

**Comment by Zamrod** [comment]
I am currently using the latest versions and it is not working:
![image](https://github.com/Aedif/tactical-grid/assets/12570777/c66cfd52-f815-4c02-839f-2fa443a06942)

**Comment by Aedif** [comment]
Contact the developer of those modules. It's a conflict between them. TG is not causing this, it's merely calling on ATC.

**Comment by Zamrod** [comment]
Done.

**Comment by caewok** [comment]
Fixed in ATV v0.6.1 and ATC v0.6.3.
